test/integration/scheduler/framework_test.go

```diff
@@ -563,6 +564,12 @@ func TestPrefilterPlugin(t *testing.T) {
 
 // TestScorePlugin tests invocation of score plugins.
 func TestScorePlugin(t *testing.T) {
+	// Create a plugin registry for testing. Register only a score plugin.
+	scPlugin := &ScorePlugin{}
```

Comment:
not your fault, but I dislike those short plugin names "scPlugin" etc. not descriptive and confusing, perhaps a followup cleanup PR to rename all of them properly, like s/scPlugin/scorePlugin, s/perPlugin/permitPlugin etc?

Reply:
Also addressed in this PR. PTAL